spring-cloud-alibaba-starters/spring-cloud-starter-alibaba-nacos-discovery/pom.xml

```diff
@@ -103,6 +107,7 @@
         <dependency>
             <groupId>org.mockito</groupId>
             <artifactId>mockito-inline</artifactId>
+            <version>${mockito-inline.version}</version>
             <scope>test</scope>
         </dependency>
```

Comment:
This dependency may no longer be needed, you can try deleting it

Reply:
It was indeed like this in Spring Boot before, but when I upgraded Spring Boot 3.2. x and Spring Cloud 2023, it seemed to have been removed because it could not pass through my Maven, which caused me to add a version

Reply:
![1702392017059](https://github.com/alibaba/spring-cloud-alibaba/assets/31510505/1eaf36d6-f90e-4970-8462-ab5fb26dd8ba)
![1702392043093](https://github.com/alibaba/spring-cloud-alibaba/assets/31510505/ba519ac1-6dd5-4f1b-8f91-964a716657c2)
After verification, it is indeed like this

Reply:
Try remove the 'mockito-inline' module dependency, it may no longer be needed

Reply:
I'm very sorry, I misunderstood your meaning at first, and now I have tried to delete it